ui/packages/consul-ui/app/components/consul/auth-method/binding-view/index.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="consul-auth-method-binding-view">
+{{#if (gt @items.length 0)}}
+  <p>Binding rules allow an operator to express a systematic way of automatically linking roles and service identities to newly created tokens without operator intervention.<br>
+Successful authentication with an auth method returns a set of trusted identity attributes corresponding to the authenticated identity. Those attributes are matched against all configured binding rules for that auth method to determine what privileges to grant the the Consul ACL token it will ultimately create.</p>
```

Comment:
```suggestion
Successful authentication with an auth method returns a set of trusted identity attributes corresponding to the authenticated identity. Those attributes are matched against all configured binding rules for that auth method to determine what privileges to grant the Consul ACL token it will ultimately create.</p>
```